src/commands/Misc/reddituser.ts

```diff
@@ -0,0 +1,233 @@
+import { CommandStore, KlasaMessage, Timestamp } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { fetch, roundNumber, getColor, cutText } from '../../lib/util/util';
+import { Reddit } from '../../lib/types/definitions/Reddit';
+import { MessageEmbed } from 'discord.js';
+import { BrandingColors } from '../../lib/util/constants';
+
+const COMPLEXITY_LEVELS = ['very low', 'low', 'medium', 'high', 'very high', 'very high'];
+
+export default class extends SkyraCommand {
+
+	private comments: Reddit.CommentDataElement[];
```

Comment:
You should really not set this as a class property... If two users run it at the same time, you can get information clashing...